### PR TITLE
Add Juno v0.4 downloads

### PR DESCRIPTION
**Pull request by one-more-minute**
If anyone has a chance to test these out quickly before we do a proper release, it'd be much appreciated.

### CODE REVIEW

File: downloads/index.md [hunk 1]
```diff
@@ -59,6 +59,26 @@ recommended that you download the v0.4.x binaries to try out Julia,
 unless you are working with code that was developed specifically for a
 previous release.
 
+
+# Julia + Juno IDE bundles (v0.3.12)
```
Comment (tkelman) [inline downloads/index.md]: wrong version number?

**Comment by ViralBShah** [comment]
I get a bunch of warnings in the julia-LT plugin:

````
WARNING: Base.String is deprecated, use AbstractString instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.String is deprecated, use AbstractString instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.String is deprecated, use AbstractString instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.String is deprecated, use AbstractString instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.String is deprecated, use AbstractString instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.String is deprecated, use AbstractString instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint32 is deprecated, use UInt32 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint8 is deprecated, use UInt8 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint32 is deprecated, use UInt32 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint32 is deprecated, use UInt32 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint32 is deprecated, use UInt32 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint32 is deprecated, use UInt32 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint32 is deprecated, use UInt32 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint32 is deprecated, use UInt32 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint8 is deprecated, use UInt8 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint32 is deprecated, use UInt32 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint32 is deprecated, use UInt32 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint32 is deprecated, use UInt32 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint32 is deprecated, use UInt32 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint8 is deprecated, use UInt8 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint8 is deprecated, use UInt8 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint32 is deprecated, use UInt32 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint32 is deprecated, use UInt32 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint32 is deprecated, use UInt32 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint32 is deprecated, use UInt32 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint32 is deprecated, use UInt32 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint32 is deprecated, use UInt32 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint32 is deprecated, use UInt32 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint32 is deprecated, use UInt32 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint32 is deprecated, use UInt32 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint8 is deprecated, use UInt8 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint8 is deprecated, use UInt8 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint32 is deprecated, use UInt32 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint32 is deprecated, use UInt32 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint32 is deprecated, use UInt32 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint32 is deprecated, use UInt32 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint32 is deprecated, use UInt32 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint32 is deprecated, use UInt32 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint8 is deprecated, use UInt8 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint16 is deprecated, use UInt16 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint32 is deprecated, use UInt32 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint64 is deprecated, use UInt64 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint128 is deprecated, use UInt128 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint8 is deprecated, use UInt8 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint16 is deprecated, use UInt16 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint32 is deprecated, use UInt32 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint64 is deprecated, use UInt64 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.Uint128 is deprecated, use UInt128 instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.String is deprecated, use AbstractString instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.String is deprecated, use AbstractString instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
WARNING: Base.String is deprecated, use AbstractString instead.
  likely near /Users/viral/Desktop/Juno.app/Contents/Resources/app/plugins/Julia-LT/jl/init.jl:45
````

**Comment by one-more-minute** [comment]
@ViralBShah does `Pkg.checkout("JuliaParser")` help at all?

**Comment by ViralBShah** [comment]
Yes, I think that is what I had done the last time, which had fixed it. Perhaps we need to tag a new version and such? Cc: @jakebolewski

**Comment by ViralBShah** [comment]
Confirming that this is not a problem with the master of JuliaParser. Probably just needs tagging. Also cc @tkelman

**Comment by one-more-minute** [comment]
As a temporary fix, the current downloads actually bundle JuliaParser master.

**Comment by ViralBShah** [comment]
Should I download the Juno linked here again to test this?

**Comment by jakebolewski** [comment]
I can tag another release.  On that note, could we move `JuliaParser` to JuliaLang or some other public organization?  I won't be writing much Julia code in the future and won't be able to maintain it

**Comment by ViralBShah** [comment]
I am not sure moving it to JuliaLang makes sense unless there is someone else who is likely to maintain it, but it can certainly go there until we figure out the right place for the package.

**Comment by ScottPJones** [comment]
That's very depressing to hear, @jakebolewski, you are a major benefit to the Julia community.
I can only hope that you will be able to come back to Julia in the future.

**Comment by one-more-minute** [comment]
@ViralBShah you probably have the same Juno bundle already, but you won't see the packages if you already have a `~/.julia/v0.4` folder.

**Pull request merged**